chia/util/write_file.py

```diff
@@ -0,0 +1,24 @@
+import os
+import shutil
+
+from pathlib import Path
+from typing import Union
+
+
+def write_file(file_path: Path, data: Union[str, bytes], *, file_mode: int = 0o600):
```

Comment:
aren't there named constants for `0o600`?

I also think there should be unit tests for this function

Reply:
Tests added.

Re: The named constants for the permission mask. I strongly prefer the octal representation, which I think is fairly universally understood by devs. The alternative is to explode out a bitmask of similarly named constants like `stat.S_IRUSR | stat.S_IWUSR |  stat.S_IRGRP | stat.S_IWGRP` etc. I find it far more readable to use the octal representation, and less prone to accidentally choose the wrong permission bit(s). The octal representation is interoperable with the named constants.